.github/workflows/adex-cli.yml

```diff
@@ -33,6 +33,19 @@ jobs:
         if: github.event_name != 'pull_request'
         run: echo "COMMIT_HASH=$(git rev-parse --short=7 HEAD)" >> $GITHUB_ENV
 
+      - name: Set up cargo cache
+        uses: actions/cache@v3
+        continue-on-error: false
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/            
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: ${{ runner.os }}-cargo-
+
```

Comment:
Can this be made into a reusable workflow instead of repeating it in multiple places?

Reply:
We can use composite actions for that, will do it